plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcConnectorTest.java

```diff
@@ -271,11 +275,13 @@ protected void verifyAddNotNullColumnToNonEmptyTableFailurePermissible(Throwable
         assertThat(e).hasMessageContaining("NULL not allowed for column");
     }
 
+    @org.junit.jupiter.api.Test
+    @Test(enabled = false) // this is a JUnit test
```

Comment:
Why do you need this?

Reply:
to disable testNG running this test. If the class has a top-level `@Test(singleThreaded = true)` it will execute alll of the `public void something()` methods.